compiler.rst

```diff
@@ -7,23 +7,14 @@ Design of CPython's Compiler
 Abstract
 --------
 
-Historically (through 2.4), compilation from source code to bytecode
-involved two steps:
+In CPython, the compilation from source code to bytecode involves three steps:
```

Comment:
From the original, it should be 'involves four steps' and 'the latter three steps'.  I intended those two phrasings to be kept.
Note 1. I am aware that this is not a general review of content, just of markup, but this opening was too annoying to ignore.  I intentionally did not look at content further on ;-).
Note 2. I have not reviewed rst markup (have forgotten too much).  Hope someone else will, and say so.